services/src/test/scala/cromwell/services/womtool/impl/WomtoolServiceInCromwellActorSpec.scala

```diff
@@ -132,6 +148,48 @@ class WomtoolServiceInCromwellActorSpec extends ServicesSpec("Womtool") {
         description = WorkflowDescription(valid = false, errors = List("Required workflow input 'wf_hello.hello.addressee' not specified"))))
     }
 
+    "return invalid for a valid workflow with empty inputs" in {
+
+      val wsfc = wsfcConjurer(workflowSource = Option(TestData.wdlValid), inputsJson = TestData.emptyInputs)
+
+      check(DescribeRequest(wsfc), DescribeSuccess(
+        description = WorkflowDescription(valid = false, errors = List("Required workflow input 'wf_hello.hello.addressee' not specified"))))
+    }
+
+    "return valid for a valid no-inputs workflow with empty inputs" in {
+
+      val wsfc = wsfcConjurer(workflowSource = Option(TestData.wdlValidNoInputs), inputsJson = TestData.emptyInputs)
+
+      check(
+        DescribeRequest(wsfc),
+        DescribeSuccess(
+          description = WorkflowDescription(
+            valid = true,
+            errors = List.empty,
+            name = "wf_hello",
+            inputs = List.empty,
+            outputs = List.empty,
+            images = List.empty,
+            submittedDescriptorType = Map(
+              "descriptorType" -> "WDL",
+              "descriptorTypeVersion" -> "1.0"
+            ),
+            importedDescriptorTypes = List.empty,
+            meta = Map.empty,
+            parameterMeta = Map.empty
+          )
+        )
+      )
+    }
+
+    "return invalid for a valid no-inputs workflow with extraneous inputs" in {
```

Comment:
Technically, I think this is only asserted in 1.0 (there are some cases where people use made up inputs as "comments" in their inputs files.

Reply:
Thank you and TIL